drivers/serial/uart_native_tty_bottom.c

```diff
@@ -175,6 +240,37 @@ static inline void native_tty_data_bits_set(struct termios *ter,
 	ter->c_cflag |= data_bits_to_set;
 }
 
+/**
+ * @brief Get the number of data bits from the termios structure
+ *
+ * @param ter
+ * @param data_bits
+ *
+ * @return 0 on success, -1 if unsupported
```

Comment:
Note all these new statics just error out on errors, so not really returning -1.